metaci/build/models.py

```diff
@@ -660,6 +660,15 @@ def run_flow(self, project_config, org_config):
 
         flow_config = project_config.get_flow(self.flow)
 
+        # If it's a release build, pass the dates in
+        options = {}
+        if self.build.plan.role == "release" and self.build.release:
+            options["github_release_notes"] = {
+                "sandbox_date": self.build.release.sandbox_push_date,
+                "production_date": self.build.release.production_push_date,
+                "trial_info": self.build.release.trialforce_id,
```

Comment:
```suggestion
```

There's no reason to include this one. The trialforce id will never be known when the release is being uploaded, because it needs to be uploaded before it can be installed into the TSO.